modules/admin_manual/pages/issues/impersonate_users.adoc

```diff
@@ -31,37 +32,36 @@ see.
 [[ending-an-impersonation]]
 == Ending an Impersonation
 
-When you’re ready to stop impersonating the user, log out and you will
+When you're ready to stop impersonating the user, log out and you will
 return to your normal user session.
 
-[[restrict-impersonation-to-groups-group-administrators]]
-== Restrict Impersonation to Groups & Group administrators
+== Allow Some or All Group Administrators To Impersonate Users
 
-As a security measure, the application lets ownCloud administrators
-restrict the ability to impersonate users to administrators of specific
-groups. When enabled and configured, only a group’s administrator can
-impersonate members of their group.
+As a security measure, the application lets ownCloud administrators restrict the ability to impersonate users to:
 
-For example, if an ownCloud administrator restricts user impersonation
-only to the group: `group1', then *only* `group1'’s administrators can
-impersonate users belonging to `group1'. No other users can impersonate
-other users.
+- All group administrators. 
+- Specific group administrators. 
 
-NOTE: ownCloud administrators can always impersonate all users of an ownCloud instance 
-when the application is installed.
+[NOTE] 
+====
+By default, when the Impersonate app is installed, only the ownCloud administrator will be allowed to impersonate users.
+When the app is installed and configured, ownCloud administrators retain the ability to impersonate all users of an ownCloud instance.
+====
 
-To enable this option, in the administrator settings panel
-(`administrator -> Settings -> Admin`) in the "**User Authentication**"
-section, you’ll see a section titled: "**Impersonate Settings**"; which you can see below.
+When enabled and configured, only a group's administrator can impersonate members of their group.
+For example, if an ownCloud administrator restricts user impersonation only to the group: `group1`, then only `group1`'s administrators can impersonate users belonging to `group1`. 
+No other users can impersonate other users.
+
+To configure it, in the administrator settings panel, which you can find under menu:administrator[Settings > Admin > User Authentication], you'll see a section titled: "**Impersonate Settings**" (which you can see below).
 
 image:apps/impersonate/impersonate-settings.png[Impersonate App settings]
 
-Check the checkbox under that, and a textbox will appear. If you click
-in the textbox, you will see a list of available groups on your ownCloud
-installation. As you type, the list will filter down to only ones that
-match the text entered, as you can see below.
+If you want to allow group admins to users within groups they administer, click btn:[Allow all group admins to impersonate users within the groups they are admins of].
```

Comment:
I might be wrong here. But I was wondering shouldn't this be rephrased as:
`If you want to allow all group admins to impersonate users within group they administer...`. If my thinking/observation is wrong, then let me know I will approve the PR right away. Thnaks.

Reply:
You're correct. I'll fix my mistake.

Reply:
Update made, @sharidas.